doc/tutorial/02_layout/index.md

```diff
@@ -157,6 +162,15 @@ Therefore, we recommend not deleting the files including the built indexes.
 To create an index and thus a layout, the individual samples of the data set are chopped up into k-mers and determine in
 their so-called bin the specific bit setting of the Bloom Filter by passing them through hash functions. This means that
 a k-mer from sample `i` marks in bin `i` with `j` hash functions `j` bits with a `1`.
+
+Example:
+Query ACGT with kmers ACG, CGT.
+Sample 1: AATGT, Sample 2: ACCGT, Sample 3: ACGTA
+2 Hash funktions
+-> Bins 1 to 3 for ACG could look like |0000|0000|0101|
+   Bins 1 to 3 for CGT could look like |0000|0110|1100|
+-> The query seems to match Sample 3
+
```

Comment:
@eseiler do you have an image for this?

Reply:
Is this the first time, we explain how this work? Otherwise, I would just reference the other part.

This also seems a bit too detailed?
If we do it here, we could also think about just explaining a plain Bloom Filter. The principle is the same.

Reply:
I added this example, because of this discussion: https://github.com/seqan/raptor/pull/201#discussion_r1047309453
We have also a really short explanation of a BF and IBF here: https://github.com/seqan/raptor/blob/main/doc/tutorial/03_index/index.md#general-idea--main-parameters.
Should we merge these explanations? Or would you leave just this one out as its too detailed?